pkg/cmd/gist/list/http.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/shurcooL/graphql"
 )
 
-func listGists(client *http.Client, hostname string, limit int, visibility string) ([]shared.Gist, error) {
+func ListGists(client *http.Client, hostname string, limit int, visibility string) ([]shared.Gist, error) {
```

Comment:
If this is going to be shared it should be moved to `pkg/cmd/gist/shared/shared.go`

Reply:
Already moved to correct location.